daemon/build_request.go

```diff
@@ -65,6 +65,10 @@ func BuildHttpTransaction(build HttpTransactionConfig) *HttpTransaction {
 		Auth:          auth,
 		InjectHeaders: injectHeaders,
 	})
+	if err != nil {
```

Comment:
This method should not fail, ever. It's why it has no error exposed on it, there is no way to recover from it.